game.py

```diff
@@ -63,23 +34,39 @@ def get_letter_from_user(snowman_word_dict, wrong_guesses_list):
             valid_input = True
 
     return user_input_string
-    
+
+def get_word_progress(snowman_word, snowman_word_dict):
+    for letter in snowman_word:
+        if not snowman_word_dict[letter]:
+            return False
+    return True
+
 def build_word_dict(snowman_word):
-    """This function takes snowman_word as input and returns 
-    a dictionary with a key-value pair for each letter in 
-    snowman_word where the key is the letter and the value is `False`.
-    """
     snowman_word_dict = {}
     for letter in snowman_word:
         snowman_word_dict[letter] = False
     return snowman_word_dict
+
+
+def print_snowman(wrong_guesses_count):
+    for i in range(SNOWMAN_WRONG_GUESSES - wrong_guesses_count, SNOWMAN_WRONG_GUESSES):
+        print(SNOWMAN_GRAPHIC[i])
+        if i == 0:
+            print(SNOWMAN_GRAPHIC[0])
+        elif i == 1:
+            print(SNOWMAN_GRAPHIC[1])
+        elif i == 2:
+            print(SNOWMAN_GRAPHIC[2])
+        elif i == 3:
+            print(SNOWMAN_GRAPHIC[3])
+        elif i == 4:
+            print(SNOWMAN_GRAPHIC[4])
+        elif i == 5:
+            print(SNOWMAN_GRAPHIC[5])
+        elif i == 6:
+            print(SNOWMAN_GRAPHIC[6])
```

Comment:
👀 These lines can be removed since you switched over to the array-based image data. This is currently causing the lines to be doubled when displaying the snowman.